assets/scss/7-layout/_grid.scss

```diff
@@ -2,9 +2,13 @@
 //
 // This is a flexbox based grid. In the future, we'll start to rely more on CSS grid and slowly phase out this grid, piece by piece.
 //
-// _Replacements so far:_
-// grid_separator => **has-<size>-btm-marg** |
-// grid_container => **l-container**
+// _Replacements so far:_<br>
+// grid_separator => [**has-{size}-btm-marg**](/pages/utilities/index.html#margin-has-size-btm-marg) <br>
+// grid_container => [**l-container**](#container-l-container) <br>
+// grid_padded => [**has-gutter-padding**](/pages/utilities/index.html#padding-has-padding) <br>
+// grid_padded--temp => [**has-temp-gutter-padding**](/pages/utilities/index.html#padding-has-padding) <br>
+// col_adunit300x250 => Removed <br>
```

Comment:
Just to confirm, this won't break non-story pages that rely on this, right? There are several that do.

Reply:
In general there are only a [few pages](https://github.com/texastribune/queso-ui#steps-to-test-breaking-changes) that go all in on this trimmed grid. `grid_padded` and `grid_padded--temp` are used on those three pages. I have a [secondary PR](https://github.com/texastribune/texastribune/pull/3605) in /texastribune to fix that once we settle this one.